test/Microsoft.Extensions.Localization.Tests/ResourceManagerStringLocalizerTest.cs

```diff
@@ -105,7 +105,7 @@ public void ResourceManagerStringLocalizer_GetAllStrings_MissingResourceThrows(b
             });
             var expected = includeParentCultures
                 ? "No manifests exist for the current culture."
-                : "The manifest 'testington.en-US.resources' was not found.";
+                : $"The manifest 'testington.{CultureInfo.CurrentCulture.ToString()}.resources' was not found.";
```

Comment:
Shouldn't need the `ToString()`
If it's important to keep `en-us` you could use `ReplaceCulture` (but this looks fine as-is).